src/Input.js

```diff
@@ -72,11 +73,16 @@ class Input extends React.Component {
       }
     }
 
+    if (['lg', 'sm'].indexOf(attributes.size) > -1) {
```

Comment:
You should also delete size from attributes so it doesn't get passed down to the DOM in this case.